tests/protocol/test_wire.py

```diff
@@ -162,16 +166,46 @@ def test_get_host_ga_plugin(self, *args):
     def test_download_ext_handler_pkg_fallback(self, patch_request, patch_get_host, patch_http, *args):
         ext_uri = 'extension_uri'
         host_uri = 'host_uri'
+        destination = 'destination'
         patch_get_host.return_value = HostPluginProtocol(host_uri, 'container_id', 'role_config')
         patch_request.return_value = [host_uri, {}]
 
-        WireProtocol(wireserver_url).download_ext_handler_pkg(ext_uri)
+        WireProtocol(wireserver_url).download_ext_handler_pkg(ext_uri, destination)
 
         self.assertEqual(patch_http.call_count, 2)
         self.assertEqual(patch_request.call_count, 1)
         self.assertEqual(patch_http.call_args_list[0][0][1], ext_uri)
         self.assertEqual(patch_http.call_args_list[1][0][1], host_uri)
 
+    @skip_if_predicate_true(running_under_travis, "Travis unit tests should not have external dependencies")
+    def test_download_ext_handler_pkg_stream(self, *args):
+        ext_uri = 'https://dcrdata.blob.core.windows.net/files/packer.zip'
+        tmp = tempfile.mkdtemp()
+        destination = os.path.join(tmp, 'test_download_ext_handler_pkg_stream.zip')
+
+        success = WireProtocol(wireserver_url).download_ext_handler_pkg(ext_uri, destination)
```

Comment:
qq - Do we need a separate test for protocol.metadata.download_ext_handler_pkg()? Underneath both the versions use restutil.http_get() so most likely no.

Reply:
We are not great about having Azure Stack tests in general. In this case I would say there is no change in behavior, so it is not worth addressing as part of this PR.

Reply:
Got it.

Thanks for doing this fix! :)